docs/installation.rst

```diff
@@ -20,6 +20,38 @@ There are different ways to install |project_name|:
     have the latest code or use revision control (each release is
     tagged).
 
+.. _installation-requirements:
+
+Pre-Installation Considerations
+-------------------------------
+
+|project_name| Repository Filesystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- No special filename requirements exist. Filenames may contain any valid
+  ascii character and be of any length. |project_name| stores data only 3
+  directory levels deep and uses short directory and filenames.
+- No special file metadata requirements exist regarding permissions or extended
+  file attributes. |project_name| only requires read/write permissions within
+  the repository filesystem.
+- Segment files generated by |project_name| are no larger than 5MB by default
```

Comment:
ok I see what you're saying.  I meant "5MB are the default segment size". That was based off this statement:
```no special requirements for the file size (borg creates small to medium sized files). see repo config, default for 1.0.x is ~5MB```
I understand the meaning was that file size limits are not a problem for Borg -- in reference to some file systems for e.g. that cannot have files larger than 2 or 4 gb.
I plan on going through these comments shortly and seeing where I can be more clear.
Appreciate the constructive criticism! :)